docs/computing/systems-roihu.md

```diff
@@ -0,0 +1,39 @@
+# Technical details about Roihu
+
+!!! note
+    This page contains preliminary information about CSC's next national
+    supercomputer Roihu, which is projected to be in researchers' use early
+    2026. Please note that the details may evolve over time.
+
+## Compute
+
+**Roihu** will have a total of 486 CPU nodes and 132 GPU nodes. The
+high-performance LINPACK (HPL) performance is estimated to be 10.5 PFlop/s for
+the CPU nodes and 23.4 PFlop/s for the GPU nodes, resulting in an aggregate HPL
+performance of 33.9 PFlop/s for the full system.
+
+The CPU nodes will have two 192-core AMD Turin CPUs each, amounting to 186 624
+CPU cores altogether. The CPUs are based on the AMD Zen 5 architecture, which
```

Comment:
72 nodes with 1536 GiB mem
414 nodes with  768 GiB

Reply:
Contrary to puhti and Mahti all gpu and cpu nodes are equipped with small nvme drives, with a size of slightly less than 1 TiB. Useful for temporary files, and software distribution.